src/components/UpgradeTableOfContents.tsx

```diff
@@ -145,11 +115,28 @@ const UpgradeTableOfContents: React.FC<IProps> = ({
   }
 
   return (
-    <Aside className={className}>
-      <OuterList>
+    <Box
+      as="aside"
+      className={className}
+      p={0}
+      mb={8}
+      textAlign="end"
+      overflowY="auto"
+      display={{ base: "none", l: "block" }}
+    >
+      <UnorderedList
+        m={0}
+        py={0}
+        ps={4}
+        pe={1}
+        fontSize="1.25rem"
```

Comment:
@pettinarip I couldn't find anything in the design system for 20px fonts.

Reply:
In this case, we should use `xl`.

`xl === 1.25rem` Here you can see the different values for `fontSize` https://chakra-ui.com/docs/styled-system/theme#typography